usfm_tools/support/parseUsfm.py

```diff
@@ -336,7 +338,14 @@ def clean(unicodeString):
     # We need to clean the input a bit. For a start, until
     # we work out what to do, non breaking spaces will be ignored
     # ie 0xa0
-    return unicodeString.replace(u'\xa0', ' ')
+    ret_value = unicodeString.replace(u'\xa0', ' ')
+
+    # escape illegal USFM sequences
+    ret_value = ret_value.replace(u'\\ ',  "\\\\ ")
+    ret_value = ret_value.replace(u'\\\n', "\\\\\n")
+    ret_value = ret_value.replace(u'\\\r', "\\\\\r")
+    ret_value = ret_value.replace(u'\\\t', "\\\\\t")
```

Comment:
This file is importing `unicode_literals` so all strings are unicode, no need for `u'`.